plugins/modules/grafana_folder.py

```diff
@@ -66,6 +74,20 @@
       title: "grafana_working_group"
       state: present
 
+- name: Create a folder and set permissions
+  community.grafana.grafana_folder:
+      url: "https://grafana.example.com"
+      grafana_api_key: "{{ some_api_token_value }}"
+      title: "grafana_working_group"
+      state: present
+      permissions:
+        - role: "Editor"
+          permission: 2
+        - role: "Viewer"
+          permission: 1
+        - userId: 2
+          permission: 4
```

Comment:
I think that would be easier for users to user permission names (view, edit, admin) instead of ids.
Same for teamId vs team name and userId vs username

Reply:
You can now use view, edit, admin for `permission `